lib/util/binarySearch.js

```diff
@@ -0,0 +1,35 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Peter Somogyvari @petermetz
+*/
+
+"use strict";
+
+/**
+ * Author: Luca Kiebel
+ * LICENSE: CC BY-SA 4.0
```

Comment:
We can't use code under `CC BY-SA 4.0` license

Reply:
> We can't use code under `CC BY-SA 4.0` license

:/ Alright, I'll look for a different implementation then.

Reply:
@evilebottnawi Got rid of it in the latest revision (now it's an MIT licensed package instead): https://github.com/webpack/webpack/commit/5fa8e1581979d9365b2523ad273863e8f3a70ead